docs/config-file/v1.rst

```diff
@@ -1,12 +1,15 @@
-Read the Docs YAML Config
-=========================
+Configuration File V1
+=====================
 
 Read the Docs now has support for configuring builds with a YAML file.
-The file, ``readthedocs.yml``, must be in the root directory of your project.
+:doc:`The Read the Docs file <index>` must be in the root directory of your project.
```

Comment:
Hi and thanks for writing docs on this very useful feature.
Maybe it would be helpful to emphasize somehow that the `yml` file has to be in the root of the repository and not in the root of the docs? That tripped me up a little (see #5042).

Something like this:
```
The file must be in the root directory of your project, **not** in the root of your `./docs` folder.

    |-- your-project
    |   |-- docs
    |   |   |-- index.md
    |   |   |-- requirements.txt
    |   |-- readthedocs.yml
    |   |-- .gitignore
    |   |-- README.md
```

Reply:
So, we have the sentence `must be in the root directory of your project` isn't that enough? Maybe we should change project for repository?